ras/hwinfo.py

```diff
@@ -55,9 +55,19 @@ def test_disk(self):
 
     def test_unique_id_save(self):
         self.run_cmd("hwinfo --disk --save-config %s" % self.Unique_Id)
+        if "failed" in process.system_output("hwinfo --disk --save-config %s"
+                                             % self.Unique_Id,
+                                             shell=True).decode("utf-8"):
+            self.log.info("--save-config UDI option failed")
```

Comment:
seems redundant as any way you logging same in self.fail

Reply:
I have removed the reduntant self.log(..) function as you said.
Thanks